### DISCUSSION TITLE
Incorrect language analysis of a C# file.

### DISCUSSION THREAD

**Discussion by TheBlackPlague** [github-linguist/linguist#5937] [Classification] [answered]
Project: https://github.com/TheBlackPlague/StockNemo
File: https://github.com/TheBlackPlague/StockNemo/blob/master/Backend/Data/PieceDevelopmentTable.cs

GitHub believes that the file is a "Smalltalk" file. It is not. It is definitely a C# file.

**Comment by lildude** [comment, marked as answer]
Looks like you're hitting https://github.com/github/linguist/issues/5870

If you've got the time and resources we'd really appreciate a PR to improve the heuristic.

Until then, you'll need to use an override to force the language.

**Reply by TheBlackPlague** [reply]
I have responded to the issue. Hopefully, my response can expedite a fix. 😄
